### ISSUE TITLE
MIDI file import

### ISSUE THREAD

**Issue by wbsoft** [wbsoft/frescobaldi#290]
We'd like to have File→Import→Midi file...

this could be implemented using Frescobaldi's own `midifile` library.

**Comment by proski** [comment]
Having GUI for tempo/precision adjustments would be very cool. Especially with a histogram of durations.

**Comment by wbsoft** [comment]
Yes and a PDF preview like the score wizard, and the ability to directly assign tracks to voices and staffs, and even auto-split large chords in single-track midi's across different voices. Indeed I proposed to do this using `midifile`, but maybe we should implement this in `ly` and have it use [Mido] (https://github.com/olemb/mido/) for MIDI reading...
